setup.py

```diff
@@ -66,6 +66,7 @@
             'annlite>=0.3.0',
             'qdrant-client~=0.6.0',
             'strawberry-graphql',
+            'elasticsearch>=8.0.1',
         ],
         'qdrant': [
             'qdrant-client~=0.6.0',
```

Comment:
maybe let's add other tags for elastic, weaviate and pqlite ?

Reply:
not for this PR I guess